Katello/katello
# Fixes #33552 - Add Breadcrumbs to all CV UI Pages

Adds breadcrumbs to all CV UI pages and tabs.
To test >
Navigate through the CV UI, details, tabs etc and verify breadcrumbs on the pages.
